README.md

```diff
@@ -164,6 +164,47 @@ async def only_content(request: Request):
     )
 ```
 
+## Usage with Litestar
+You can use Jinja2 Fragments with Litestar by using the `LitestarHTMXTemplate` class. This gives you access to the `block_name` parameter when rendering the template.
+
+By default, the full page is rendered unless you provide a `block_name` keyword argument.
+
+```py
+from pathlib import Path
+
+from litestar.contrib.htmx.request import HTMXRequest
+from litestar import get, Litestar
+from litestar.response import Template
+
+from litestar.contrib.jinja import JinjaTemplateEngine
+from litestar.template.config import TemplateConfig
+from jinja2_fragments.litestar import LitestarHTMXTemplate as HTMXTemplate
+
+
+@get('/full_page')
+def full_page(request: HTMXRequest):
+    return HTMXTemplate(
+        template='page.html.jinja2',
+        context={"magic_number": 42}
+    )
+
+def only_content(request: HTMXRequest):
+    return HTMXTemplate(
+        template='page.html.jinja2',
+        block_name='content',
+        context={"magic_number": 42}
+    )
+
+app = Litestar(
+    route_handlers=[full_page, only_content],
+    request_class=HTMXRequest,
+    template_config=TemplateConfig(
+        directory=Path("path/to/templates"),
```

Comment:
Using `Path` is good practice, but can we use as string to simplify the example and be consistent with the other examples?

Reply:
Yup, no issue here. I'll update that as well.